src/protocol/organization.rs

```diff
@@ -248,11 +290,11 @@ impl OrganizationConfigRepOk {
     }
 
     #[getter]
-    fn active_users_limit(_self: PyRef<'_, Self>) -> PyResult<Option<u64>> {
-        Ok(match _self.as_ref().0 {
+    fn active_users_limit(_self: PyRef<'_, Self>) -> PyResult<ActiveUsersLimit> {
+        Ok(match &_self.as_ref().0 {
             organization_config::Rep::Ok {
                 active_users_limit, ..
-            } => active_users_limit,
+            } => ActiveUsersLimit(active_users_limit.clone()),
```

Comment:
Maybe make a type apart in `libparsec_types`, it is used in `OrganizationConfig` (this is my current work)
then you can derive from Copy to avoid the heap

Reply:
This would create two types for `ActiveUsersLimit`? One which implements `Copy` and the other generated by the `serialization_format` macro. Is there a way to use the same type for both?

Reply:
I mean maybe we should don't implement ActiveUsersLimit in `serialization_format`, but we can keep your work

Reply:
This is weird if a type depends on `authenticated_cmds::v2::organization_config::ActiveUsersLimit`

Reply:
What should we do? Remove `ActiveUsersLimit` from `serialization_format`? It make sense to me if `organization_config` which has a `user_limit` field embed an `ActiveUsersLimit` type.

Reply:
Yes you can embed without `serialization_format`